frontend/components.js

```diff
@@ -350,31 +397,33 @@ const LoginStatus = connect(
     h(
       'div',
       {class: 'status-text-wrapper'},
-      h('span', {class: 'status-text'}, `Balance: ${balance / 1000000} ADA`),
-      h('span', {class: 'status-text'}, `ID: ${activeWalletId}`)
+      h(
+        'div',
+        {class: 'status-text'},
+        'Balance: ',
+        h('span', {class: 'status-balance'}, `${(balance / 1000000).toFixed(6)} ADA`)
+      ),
+      h(
+        'div',
+        {class: 'status-text', title: activeWalletId},
+        'WalletID: ',
+        h(CopyOnClick, {value: activeWalletId, title: activeWalletId})
```

Comment:
I think wallet ids should not be copyable to discourage people from accidentally using them as an address since it is not recommended: https://cardanodocs.com/technical/hd-wallets/ "**Crucial point in design: root public keys are not used to actually store money**." The wallet id is nothing else than the address associated with the root public key. A more detailed explanation is provided in the paragraph above in the linked article.

I think it is enough to display the address as a title on hover

Reply:
I probably like the most the second option, i.e. taking the shortened SHA2 hash as the displayed "wallet id". Shortening the hash should be fine: https://security.stackexchange.com/questions/97377/secure-way-to-shorten-a-hash?utm_medium=organic&utm_source=google_rich_qa&utm_campaign=google_rich_qa

Reply:
Well, the other thing is that dropping it completely (or moving to some less promiment place) woudl leave more place to display balance on top - which is a more valuable information, and its currently competing for space.

Reply:
Or let's just drop it completely for now. You can use as a reference some of the used addresses in the "send" tab.